modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding {product-title} repositories to Image Builder
+Use the following procedure to add the {product-title} repositories to Image Builder on your build host.
+
+.Prerequisites
+* Your build host meets the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/setting-up-image-builder_composing-installing-managing-rhel-for-edge-images#edge-image-builder-system-requirements_setting-up-image-builder[Image Builder system requirements]. 
+* You have installed and set up Image Builder and the `composer-cli` tool as described in
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/setting-up-image-builder_composing-installing-managing-rhel-for-edge-images#edge-installing-image-builder_setting-up-image-builder[Installing Image Builder].
+* You have root user access to your build host.
+
+.Procedure
+
+. Enable the {product-title} RPM repositories on the build host:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
+
+. Install the `reposync` and `createrepo` tools:
++
+[source,terminal]
+----
+$ sudo yum install -y yum-utils
+----
+
+. Sync the {product-title} RPM packages to your build host:
++
+[source,terminal]
+----
+$ sudo reposync --arch=$(uname -i) --arch=noarch --gpgcheck \
+    --download-path /var/repos/microshift-local \
```

Comment:
`/var/repos` is not a standard location. I'm not sure we should be creating top-level dirs in /var.
Suggesting to use `/var/cache/microshift/repos`, also used by osbuild, etc.

Reply:
Also, let's set repo base path in a variable to make it easier for the users to relocate if needed.
```
REPO_PATH=/var/cache/microshift/repos
```

Reply:
`/var/repo` is the location used in the RH KB article (https://access.redhat.com/solutions/23016) referenced from the RHEL docs. I didn't use `/var/cache/$app` as this is normally for data that'd be recreated by the corresponding $app, but I don't think the exact location doesn't matter, as long as it's not in a user dir and accessible by the `_osbuild` user.